### DISCUSSION TITLE
How to change in separate colors of the user, host and the same for root?

### DISCUSSION THREAD

**Discussion by BlagoYar** [romkatv/powerlevel10k#2747] [General]
Actually how to do it?

**Comment by romkatv** [comment]
How does you prompt look now? How do you want it to look instead?

**Reply by BlagoYar** [reply]
> How does you prompt look now? How do you want it to look instead?

Now
![ScreenShot - 2](https://github.com/user-attachments/assets/847aa742-0a19-42a1-9e0f-30c79626c596)

Want example:
$${\color{red}root}$$ $${\color{aqua}@}$$ $${\color{green}hostname}$$

I also want for an ordinary user for example:
$${\color{darkblue}SimpleUser}$$ $${\color{orange}@}$$ $${\color{darkred}hostname}$$

**Reply by romkatv** [reply]
Open `~/.p10k.zsh` and find this line: https://github.com/romkatv/powerlevel10k/blob/2b7da93df04acd04d84f5de827e5b14077839a4b/config/p10k-classic.zsh#L930

Change it to this:

```zsh
typeset -g POWERLEVEL9K_CONTEXT_ROOT_TEMPLATE='%B%F{1}%n%F{6}@%F{2}%m'
```

You can test these things like so:

```zsh
print -P '%B%F{1}%n%F{6}@%F{2}%m'
```

Docs: https://zsh.sourceforge.io/Doc/Release/Prompt-Expansion.html

**Reply by BlagoYar** [reply]
> You can test these things like so:

Hmmm ... Strange, but print shows the established color, and the console after the first change no longer changes it.
![ScreenShot - 3](https://github.com/user-attachments/assets/f7e43ee2-271f-4891-8a6b-a5779deb65d6)

**Reply by romkatv** [reply]
If you mean that the colors stay the same when you log in as root, you need to modify `/root/.p10k.zsh`.

**Reply by BlagoYar** [reply]
> If you mean that the colors stay the same when you log in as root, you need to modify

No, I changed the code offered by you in the ordinary user file without changes, it turned out that on the right.
Then I changed the color of the separator (on the left), but it remained unchanged.
Using the application, `stacer` cleaned the cache of applications and it seems now to work as it should.

Now everything is as I wanted
```
   typeset -g POWERLEVEL9K_CONTEXT_ROOT_TEMPLATE='%F{#FD6347}%n%F{6}%F{227}@%F{2}%m'
   typeset -g POWERLEVEL9K_CONTEXT_TEMPLATE='%F{156}%n%F{6}%F{227}@%F{2}%m'
```

![ScreenShot - 5](https://github.com/user-attachments/assets/b0a82938-32d1-49e0-bedd-c5583f3aa89a)


Thanks
